.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -37,6 +37,7 @@ This checklist is meant to remind the package maintainer(s) who will review this
 - [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
 - [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
 - [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
+- [ ] Do the commits need to be squashed or rebased? This is only important if the milestone is for bugfix because we need a clean merge commit for the backport (no "Squash and merge").
 - [ ] Did the CI pass? If not, are the failures related?
-- [ ] Is a milestone set?
+- [ ] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
```

Comment:
what it is a bugfix, how should the change log be handled?  Just put in the "unreleased" entry and the release manager will move it to the correct spot?

Reply:
Maybe @astrofrog remembers, for `astropy`, I *think* in the before-towncrier days, we used to have 2 unreleased sections in `main`/`master`, 1 for bugfix and 1 for the next major release. We first set milestone when PR comes in, then we tell the contributor which section to put things in. Then the release manager has to resolve conflict during manual backport to get that entry into the correct backport branch version of the change log. Then when bugfix is released, the bugfix section in `main`/`master` manually gets updated. ❓